editor.planx.uk/src/pages/FlowEditor/components/Settings/Submissions/EventsLog.tsx

```diff
@@ -85,30 +87,32 @@ const CollapsibleRow: React.FC<Submission> = (submission) => {
   return (
     <React.Fragment key={submission.eventId}>
       <TableRow sx={{ "& > *": { borderBottom: "unset" } }}>
-        <TableCell
-          sx={{
-            display: "flex",
-            flexDirection: "row",
-            alignItems: "center",
-          }}
-        >
-          <Avatar
+        <TableCell>
+          <Box
             sx={{
-              background: "none",
-              marginRight: (theme) => theme.spacing(1),
+              display: "flex",
+              flexDirection: "row",
+              alignItems: "center",
             }}
           >
-            {submission.eventType === "Pay" ? (
-              <Payment
-                color={submission.status === "Success" ? "success" : "error"}
-              />
-            ) : (
-              <Send
-                color={submission.status === "Success" ? "success" : "error"}
-              />
-            )}
-          </Avatar>
-          {submission.eventType}
+            <Avatar
+              sx={{
+                background: "none",
+                marginRight: (theme) => theme.spacing(1.5),
+              }}
+            >
+              {submission.status === "Success" ? (
+                <CheckCircleIcon color="success" />
+              ) : (
+                <CancelIcon color="error" />
+              )}
+            </Avatar>
+
+            {submission.eventType === "Pay" ? <Payment /> : <Send />}
+            <Typography variant="body2" ml={0.5}>
+              {submission.eventType}
+            </Typography>
+          </Box>
```

Comment:
nit: I'm not sure I'm sold on the double icon, it feels quite awkward to me. If success/failure isn't actually clear enough by red/green color alone would it be better to just add a column to the table where we print the actual word "Success" or "Failded (XXX)"? Would have additional benefit of providing additional context of status code on fail? 

![Screenshot from 2024-05-16 13-54-59](https://github.com/theopensystemslab/planx-new/assets/5132349/3b86d9f6-b8b5-4efc-9939-19aea371f5d5)

Reply:
Eg columns: "Event", "Status", "Date", "Session ID" ?